src/workspace/config/snapshot.json

```diff
@@ -247,121 +175,45 @@
         "showFavicons": true,
         "defaultFaviconUrl": "https://cdn.openfin.co/workspace/5.0.0/icons/defaultFavicon.svg"
       },
-      "includeInSnapshots": true,
+      "frame": false,
+      "icon": "",
+      "maxHeight": -1,
+      "maxWidth": -1,
+      "maximizable": true,
+      "minHeight": 200,
+      "minWidth": 200,
+      "minimizable": true,
+      "opacity": 1,
       "resizable": true,
       "resize": true,
+      "resizeRegion": {
+        "bottomRightCorner": 20,
+        "size": 7,
+        "sides": {
+          "top": true,
+          "right": true,
+          "bottom": true,
+          "left": true
+        }
+      },
       "saveWindowState": false,
       "shadow": false,
       "showTaskbarIcon": true,
       "showBackgroundImages": false,
       "smallWindow": false,
       "spellCheck": false,
-      "state": "normal",
+      "state": "maximized",
```

Comment:
do we definitely want it to maximise when it's launched?

Reply:
again, just restoring previous